test/FacebookAdsTest/AbstractTestCase.php

```diff
@@ -94,6 +94,11 @@ class AbstractTestCase extends \PHPUnit_Framework_TestCase {
   public static $testRunId;
 
   /**
+   * @var string
+   */
+  public static $httpAdapter;  
```

Comment:
i feel it will better to rename this in httpAdapterName, because i think it will be easy to mistakenly expect that $httpAdapter will hold the actual adapter instance